inference/core/env.py

```diff
@@ -634,6 +634,10 @@
     ROBOFLOW_API_REQUEST_TIMEOUT = int(ROBOFLOW_API_REQUEST_TIMEOUT)
 
 
+# Control SSL certificate verification for requests to the Roboflow API
+# Default is True (verify SSL). Set ROBOFLOW_API_VERIFY_SSL=false to disable in local dev.
+ROBOFLOW_API_VERIFY_SSL = str2bool(os.getenv("ROBOFLOW_API_VERIFY_SSL", "True"))
```

Comment:
Needed for the new `localapi` because Python doesn't respect the system keychain certificate store apparently.